diff --git a/node_modules/@vitest/mocker/dist/node.js b/node_modules/@vitest/mocker/dist/node.js
index f7dad3e..246a416 100644
--- a/node_modules/@vitest/mocker/dist/node.js
+++ b/node_modules/@vitest/mocker/dist/node.js
@@ -1284,7 +1284,9 @@ function mockerPlugin(options = {}) {
 				});
 			},
 			async load(id) {
-				if (id !== registerPath) {
+				// ignore the cache-busting query in:
+				// @vitest/mocker/dist/register.js?v=20e98b78
+				if (!id.startsWith(registerPath)) {
 					return;
 				}
 				if (!server) {
